erlang_ls.config

```diff
@@ -0,0 +1,23 @@
+# for linux
+otp_path: "/home/max/.asdf/installs/erlang/24.1.2"
```

Comment:
Your personal configs should not be committed :)

Reply:
Yup. My bad.